assembly/overworld_scripts/dungeons/varisi_forest.s

```diff
@@ -27,3 +28,44 @@ EventScript_VarisiForest_LassMimi:
     trainerbattle0 0x0 0xb 0x0 gText_VarisiForest_LassMimi_Intro gText_VarisiForest_LassMimi_Defeat
     msgbox gText_VarisiForest_LassMimi_Chat MSG_NORMAL
     end
+
+.global EventScript_VarisiForest_TrainerTip
+EventScript_VarisiForest_TrainerTip:
+    npcchat gText_VarisiForest_TrainerTip
+    end
+
+.global EventScript_VarisiForest_PoisonWarning
+EventScript_VarisiForest_PoisonWarning:
+    lock
+    faceplayer
+    msgbox gText_VarisiForest_PoisonWarning MSG_NORMAL
+    checkflag 0x230 @ Has received Antidote gift
+    if NOT_SET _goto EventScript_VarisiForest_AntidoteGift
+    goto EventScript_VarisiForest_PoisonWarningEnd
+
+EventScript_VarisiForest_AntidoteGift:
```

Comment:
dont need .global EventScript_VarisiForest_AntidoteGift ?

Reply:
No, it is only called by another script.  .global is required for scripts we want to bind to an event in the eventscripts file